packages/components/src/InputNumber/index.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import React from 'react'
+import { IoIosAdd, IoIosRemove } from 'react-icons/io'
+import styled from '@emotion/styled'
+import { css } from '@emotion/core'
+import theme from '@deipnon/themes/lib'
+
+const inputNumberSpanStyle = css`
+ color: ${theme.colors.azureRadiance};
+ user-select: none;
+`
+
+const inputNumberLabelStyle = css`
+color: ${theme.colors.darkGray};
+cursor: pointer;
+`
+const inputNumberFormStyle = css`
+ font-weight: 400;
+ width: 75px;
+ min-width: 75px;
+ display: flex;
+ justify-content: space-between;
+ border-radius: 1px;
+ border: 1px solid ${theme.colors.lightGrayishBlue};
+ background-color: ${theme.colors.white};
+ line-height: 20px;
+ align-items: center;
+`
+
+const InputNumberForm = styled('div')(inputNumberFormStyle)
+const InputNumberSpanElement = styled('span')(inputNumberSpanStyle)
+const InputNumberButtonElementAdd = styled(IoIosAdd)(inputNumberLabelStyle)
+const InputNumberButtonElementRemove = styled(IoIosRemove)(inputNumberLabelStyle)
+
+type PropsType = {
+	max: number,
+	min: number,
+	onChange: (e: number) => number,
+	value: number
+}
+
+const InputNumber = (props: PropsType) => {
```

Comment:
```
const InputNumber = ({ 
 value,
 min,
 max,
 onChange
}: PropsType) => {
```
Bi alt satiri silebilirsin

Reply:
@melihkorkmaz bu bahsettiğin satırı sildiğimde sürekli hata aldım ve çözemedim sorunu. Bu sebeple bu değişikliğe sonra bakacağım. https://github.com/Deipnon/deipnon/pull/59/commits/5a079ff9330ea77354b8bd5332a2f565faa82490 bu committe bu değişiklik bu sebeple şu an yok.  @mksglu ile birlikte bakacağız.